assets/tests/components/mapPage.test.tsx

```diff
@@ -89,10 +89,6 @@ jest.mock("../../src/hooks/useStations", () => ({
   useStations: jest.fn(() => []),
 }))
 
-afterEach(() => {
-  jest.restoreAllMocks()
-})
-
```

Comment:
Identifying which mocks were resetting and what the indented behavior was ended up feeling like a large task, given that these `restoreAllMocks` calls were intended to be a patch until we implemented https://app.asana.com/0/1152340551558956/1203482629607208/1204398183751515, I think we can (unfortunately) punt this till we pickup the mock reset ticket.